Music.py

```diff
@@ -19,6 +19,58 @@
 AUDIOBANK_DMADATA_INDEX: int = 3
 AUDIOTABLE_DMADATA_INDEX: int = 5
 
+bgmlist = [
```

Comment:
Unless I am missing something, aren't all these lists just duplicated data from the `*_sequence_ids` tuples? You should iterate over the tuples instead and just get the id from there instead of duplicating them.

Reply:
There's one value in bgmlist that isn't in the other list for special handling with the fairy fountain but yea I probably should have done that but I didn't have a lot of python knowledge at the time I kind had to learn for this fix as its my first time messing with anything ootr.